plugins/scaffolder/src/components/fields/RepoUrlPicker/BitbucketRepoPicker.tsx

```diff
@@ -21,17 +21,36 @@ import InputLabel from '@material-ui/core/InputLabel';
 import { Select, SelectItem } from '@backstage/core-components';
 import { RepoUrlPickerState } from './types';
 
+/**
+ * The underlying component that is rendered in the form for the `BitbucketRepoPicker`
+ * field extension.
+ *
+ * @public
+ * @param allowedOwners - Allowed workspaces for the Bitbucket cloud repository
```

Comment:
allowed WORKSPACES, or *owners* of workspaces?  wonder if that should be clarified? (though to be fair, I'm not an active user of BitBucket, so defer to your expertise :) )